feat: uniform layout and naming of hints, remarks, exceptions (#660) (#860)
zalando/restful-api-guidelines: chapters/json-guidelines.adoc

The first character must be a lower case letter, or an underscore, and subsequent
characters can be a letter, an underscore, or a number.

[EXAMPLE]

> ====

Reviewer: ```suggestion
```

[source]
----
customer_number, sales_order_number, billing_address
----

> ====

Reviewer: ```suggestion
```

- `created_at`, `modified_at`, `occurred_at`, `returned_at`  -- instead of `created`, `modified`, ...
- `campaign_start_time`, `arrival_date`, `checkout_time`  -- instead of `campaign_start`, `arrival`, ...

> TIP: Use `format: date-time` or `format: date` as required in <<238>>.

Reviewer: ```suggestion
NOTE: Make sure to use standard date-time formats like`format: date-time` or `format: date` as required in <<238>>.
```
